WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -467,12 +481,20 @@ private extension NewOrderViewModel {
 
                 let shippingMethodTitle = order.shippingLines.first?.methodTitle ?? ""
 
-                let orderTotal = itemsTotal.adding(shippingTotal)
+                let feesTotal = order.fees
+                    .map { $0.total }
+                    .compactMap { self.currencyFormatter.convertToDecimal(from: $0) }
+                    .reduce(NSDecimalNumber(value: 0), { $0.adding($1) })
+
+
+                let orderTotal = itemsTotal.adding(shippingTotal).adding(feesTotal)
```

Comment:
I just want to surface [this task](https://github.com/woocommerce/woocommerce-ios/issues/6236) that we should not be making this calculation on the VM, but reading the order total directly from the order object.

Reply:
Good point! I'll keep this change, so we can refactor the whole totals logic at once. And the calculation will be correct until then.